lib/jvm.sh

```diff
@@ -37,11 +37,24 @@ get_jdk_version() {
 }
 
 get_full_jdk_version() {
-  local jdkVersion="${1:?}"
+  # The version argument can potentially have a prefix which denotes the
+  # OpenJDK distribution. This function only normalizes the actual version
+  # and keeps the prefix intact.
+  IFS='-' read -r prefix version <<<"${1:?}"
```

Comment:
How will the new implementation handle there being more than one `-` delimiter?
eg `heroku-11.0.5-foo`

Will the `foo` be silently ignored, when it would previously have resulted in an error?

Reply:
Good comment! Luckily I had that issue in mind, it will split only the first occurence, in your example `prefix` will be `heroku` and `version` will be `11.0.5-foo`. 👍🏻